clang/test/SemaHLSL/BuiltIns/asfloat-errors.hlsl

```diff
@@ -24,12 +24,3 @@ float test_float_half(half p1) {
     // expected-note@hlsl/hlsl_detail.h:* {{candidate template ignored: could not match 'vector<half, N>' against 'half'}}
     // expected-note@hlsl/hlsl_detail.h:* {{candidate template ignored: substitution failure [with U = float, T = half]: no type named 'Type'}}
 }
-
-
-float test_float_half(bool p1) {
-    return asfloat(p1);
```

Comment:
Presumably, now that p1 is 32 bits, it can be cast to a float, so this doesn't emit any errors, right?